daemon/daemon.go

```diff
@@ -760,9 +761,30 @@ func (daemon *Daemon) Shutdown() error {
 
 				go func() {
 					defer group.Done()
-					// If container failed to exit in 10 seconds of SIGTERM, then using the force
-					if err := c.Stop(10); err != nil {
-						logrus.Errorf("Stop container %s with error: %v", c.ID, err)
+					// TODO(windows): Handle docker restart with paused containers
+					if c.IsPaused() {
```

Comment:
Some documentation here would be awesome, basically what you put in the PR comment, along the lines of why we are sending TERM first, then unpausing.